src/components/Base/Image/index.jsx

```diff
@@ -33,6 +33,10 @@ export default class Image extends React.Component {
   }
 
   shouldComponentUpdate(nextProps, nextState) {
+    if (nextProps.src !== this.props.src) {
```

Comment:
don't setState in `shouldCompnentUpdate`

Reply:
If src value initialize to "", this.state.failed will true. The nex props src has value, but 'failed' still is true, this is a bug.